notebooks/en/semantic_cache_chroma_vector_database.ipynb

```diff
@@ -0,0 +1,1389 @@
+{
```

Comment:
<p>We can assume the reader to already know pandas and numpy if they're advanced enough to be building a nontrivial RAG system. So I'd remove this explanation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>removed</p>